lib/ui/index.ts

```diff
@@ -23,7 +23,7 @@ class ShredderUI {
 
     let link = document.createElement('a');
 
-    link.download = 'scenarios.json';
+    // link.download = 'scenarios.json';
```

Comment:
@ilyagelman This fails compilation, I'm not sure how Typescript decides if something is supported or not.
Also, it's not supported in regular IE:
https://developer.mozilla.org/en/docs/Web/HTML/Element/a